src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj

```diff
@@ -10,5 +10,6 @@
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <Version>6.3.0-preview</Version>
```

Comment:
Better option remove VersionPrefix and replace it with <Version>
Add it under top property group where all package meta data is declared.